test/index.js

```diff
@@ -106,20 +84,34 @@ describe('Vero', function(){
         .track({ event: 'event' })
         .error('Bad Request', done);
     });
+
+    it('should unsubscribe when the event name is "unsubscribe"', function(done){
```

Comment:
it probably should only match exactly, but the client-side integration matches anywhere in it, so if anyone is doing something like `unsubscribe user` it would stop matching.